spec/services/earthworks_search_service_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EarthworksSearchService do
+  subject(:service) { described_class.new }
+
+  let(:response) do
+    Rails.root.join('spec/fixtures/earthworks.json').read
+  end
+
+  let(:query) { 'my query' }
+
+  before do
+    stub_request(:get, /.*/).to_return(body: response)
+  end
+
+  it { expect(service).to be_an described_class }
+  it { expect(service.search(query)).to be_an EarthworksSearchService::Response }
+
+  describe '#records' do
```

Comment:
I don't think this class has a records method and it doesn't look like this test covers it.